packages/core/npm-aqua-compiler/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "type": "module",
+  "name": "@fluencelabs/npm-aqua-compiler",
+  "version": "0.0.0",
+  "description": "Tool for converting npm imports to aqua compiler input",
+  "main": "index.js",
+  "scripts": {
+    "test": "vitest run"
+  },
+  "files": [
+    "dist"
+  ],
+  "keywords": [],
+  "author": "Fluence Labs",
+  "license": "Apache-2.0",
+  "dependencies": {
+    "@npmcli/arborist": "7.2.2",
```

Comment:
I suggest to use `^7.2.1` here cause this is what current npm is using. It's just tiny little bit more chance of everything working correctly
And we should try keeping them in sync